ShareTheTable/View Controllers/Profile Making/Add Photos/AddPhotosViewController.m

```diff
@@ -0,0 +1,137 @@
+//
+//  AddPhotosViewController.m
+//  ShareTheTable
+//
+//  Created by Andrea Gonzalez on 7/7/22.
+//
+
+#import "AddPhotosViewController.h"
+#import <Photos/Photos.h>
+#import <PhotosUI/PhotosUI.h>
+#import "SceneDelegate.h"
+#import "CreateAccountViewController.h"
+#import "QuestionViewController.h"
+#import "UniformTypeIdentifiers/UniformTypeIdentifiers.h"
+#import "Parse/Parse.h"
+
+@interface AddPhotosViewController () <PHPickerViewControllerDelegate>
+
+
+@end
+
+@implementation AddPhotosViewController
+
+- (IBAction)tapNext:(id)sender {
+    SceneDelegate *sceneDelegate = (SceneDelegate *)[UIApplication sharedApplication].connectedScenes.allObjects.firstObject.delegate;
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    
+    CreateAccountViewController *navViewController = [storyboard instantiateViewControllerWithIdentifier:@"QuestionNav"];
+    sceneDelegate.window.rootViewController = navViewController;
+}
+
+- (IBAction)tapCancel:(id)sender {
+    SceneDelegate *sceneDelegate = (SceneDelegate *)[UIApplication sharedApplication].connectedScenes.allObjects.firstObject.delegate;
+    
+    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+    
+    CreateAccountViewController *navViewController = [storyboard instantiateViewControllerWithIdentifier:@"AccountNav"];
+    sceneDelegate.window.rootViewController = navViewController;
+    
+    // TODO: DELETE ACCOUNT MADE
+}
+
+- (IBAction)tapSelectPhotos:(id)sender {
+    // self.imageViews = [[NSMutableArray alloc] init];
+    PHPickerConfiguration *config = [[PHPickerConfiguration alloc] init];
+    config.selectionLimit = 5;
+    config.filter = [PHPickerFilter imagesFilter];
+
+    PHPickerViewController *pickerViewController = [[PHPickerViewController alloc] initWithConfiguration:config];
+    pickerViewController.delegate = self;
+    [self presentViewController:pickerViewController animated:YES completion:nil];
+}
+
+- (void)viewDidLoad {
```

Comment:
It's best to organize functions based on when they are used in the flow of code. Usually viewDidLoad should be the first function. This makes it easier to follow what happens where.